src/init.cpp

```diff
@@ -415,9 +415,9 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-whitelist=<IP address or network>", _("Whitelist peers connecting from the given IP address (e.g. 1.2.3.4) or CIDR notated network (e.g. 1.2.3.0/24). Can be specified multiple times.") +
         " " + _("Whitelisted peers cannot be DoS banned and their transactions are always relayed, even if they are already in the mempool, useful e.g. for a gateway"));
 
-#ifdef ENABLE_WALLET
-    strUsage += GetWalletHelpString(showDebug);
-#endif
+    if (g_wallet_init_interface) {
+        g_wallet_init_interface->GetWalletHelpString(showDebug);
```

Comment:
This should be `strUsage += ...` I believe.

Reply:
You're right. Fixed